examples/evaluate/05-TBROM_coSimulation_pyMAPDL.py

```diff
@@ -134,23 +146,25 @@ def snapshot_to_fea(snapshot_file, geometry_file):
 ###############################################################################
 # Apply loads and boundary conditions and solve the model
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Apply loads and boundary conditions and then solve the model.
 
-# Read all nodal coords. to an array & extract the X and Y min. bounds
+# Read all nodal coordinates to an array and extract the X and Y
+# minimum bounds
 array_nodes = mapdl.mesh.nodes
 Xmax = np.amax(array_nodes[:, 0])
 Ymin = np.amin(array_nodes[:, 1])
 Ymax = np.amax(array_nodes[:, 1])
 
-# Enter /SOLU processor to apply loads and BCs
+# Enter /SOLU processor to apply loads and boundary conditions
 mapdl.finish()
 mapdl.slashsolu()
 
 # Enter non-interactive mode to assign thermal load at each node using imported data
 with mapdl.non_interactive:
     for node, temp in zip(node_num, temperature_load_val):
         mapdl.bf(node, "TEMP", temp)
-# Use the X and Y min. bounds to select nodes from five surfaces that are to be fixed and created a component and fix
-# all DOFs.
+# Use the X and Y minimum bounds to select nodes from five surfaces that are to be fixed
+# Create a component and fix all DOFs
```

Comment:
```suggestion
# Create a component and fix all degrees of freedom (DoFs)
```
The Ansys Writing Style Guide that was recently publicized in the ARC Insider newsletter says the following:
     Degrees of freedom on first mention. DoF for any subsequent mentions.